### ISSUE TITLE
Custom SurfaceMaterials

### ISSUE THREAD

**Issue by therusln** [googleprivate/resonance-audio-unity-sdk#66]
How can I access SurfaceMaterials to create my own and add properties? I'm aware that the enum with materials are on RoomManager, but couldn't find anything about their properties.

**Comment by PatrickFSLin** [comment]
I have the same issue

**Comment by amilo** [comment]
I also would like to know if there is a way to change them from Unity (maybe an overriding/writing script ?)
It would be nice if rather than recompiling the source code we could add our own to the list.
Maybe it can be helpful for you to know that their definition is here:
https://github.com/resonance-audio/resonance-audio/blob/master/platforms/common/room_effects_utils.cc

**Comment by jkammerl** [comment]
Correct, the reflection coefficients for the different materials are defined here: 
https://github.com/resonance-audio/resonance-audio/blob/2fbf9c3b77de42aef5570b08243965226b48fdf4/platforms/common/room_effects_utils.cc#L34

See also comment:
// Air absorption coefficients at 20 degrees Celsius and 50% relative humidity, according to:
http://www.music.mcgill.ca/~gary/courses/papers/Moorer-Reverb-CMJ-1979.pdf

The Resonance Audio API can be configured with an individual set of RT60s via SetRoomProperties, see here:
https://github.com/resonance-audio/resonance-audio/blob/f71b9d0ee508af724a46c87e894500eba4e23913/platforms/unity/UnityIntegration/Assets/ResonanceAudio/Scripts/ResonanceAudio.cs#L105

void SetRoomProperties(RoomProperties* room_properties, float* rt60s)
If room_properties==null, then the second argument, the set of RT60s is used to configure the reverb. You need to provide 9 RT60's for the reverberation tail at the following octave band centres: 31.25 62.5  125   250   500   1000  2000  4000  8000 Hz.

I hope this helps!
